testing/postgres-client-tests/java/PostgresTest.java

```diff
@@ -0,0 +1,118 @@
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.sql.ResultSet;
+
+public class PostgresTest {
```

Comment:
This is good, but it would be better if we also tested setting positional parameters. Maybe do a followup with that.